app/src/main/java/com/keylesspalace/tusky/fragment/NotificationsFragment.java

```diff
@@ -740,13 +744,12 @@ public void onPreferenceChanged(String key) {
                 hideFab = PreferenceManager.getDefaultSharedPreferences(getContext()).getBoolean("fabHide", false);
                 break;
             }
-            case "mediaPreviewEnabled": {
+            case "absoluteTimeView": {
```

Comment:
Not clear why this has changed? Should be both?..

Reply:
nope this was a mistake, thx for finding it